### PR TITLE
Delete handler

### PR DESCRIPTION
**Pull request by meyerjrr**
## Describe your changes

- Adds delete endpoint for handlers. Fix return type on get endpoint to be for a single type rather than list 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Issue and Documentation

-

## Testing

1.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Do we need to implement analytics? If so, have you followed up with @ellie-narducci?
- [ ] I have made corresponding changes to the documentation, docs PR has been linked.
- [ ] New and existing unit tests pass with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: pkg/api/handler.go [hunk 1]
```diff
@@ -70,3 +70,25 @@ func (a *API) AdminGetHandler(w http.ResponseWriter, r *http.Request, id string)
...
+		return
+	}
+
+	err = a.DB.Delete(ctx, q.Result)
+	if err != nil {
+		apio.Error(ctx, w, err)
+		return
+	}
+
+	apio.JSON(ctx, w, q.Result.ToAPI(), http.StatusOK)
```
Comment (JoshuaWilkes) [inline pkg/api/handler.go]:
```suggestion
	apio.JSON(ctx, w, nil, http.StatusOK)
```

File: openapi.yml [hunk 2]
```diff
@@ -1154,13 +1154,30 @@ paths:
...
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/TGHandler"
+        "401":
+          $ref: "#/components/responses/ErrorResponse"
+        "403":
```
Comment (JoshuaWilkes) [inline openapi.yml]:
```suggestion
        "404":
```

File: openapi.yml [hunk 3]
```diff
@@ -1154,13 +1154,30 @@ paths:
...
         name: id
         in: path
         required: true
+    delete:
+      summary: ""
+      operationId: admin-delete-handler
+      responses:
+        "200":
+          description: OK
+          content:
```
Comment (JoshuaWilkes) [inline openapi.yml:1170]: Don't return a body for this API

**Review by JoshuaWilkes — CHANGES_REQUESTED** [review]
I have requested that the API doesn't return a body on deletion and you also need to add API layer tests for basic functionality

File: openapi.yml (hunk 3 above)
Comment (meyerjrr) [reply openapi.yml:1170]: @JoshuaWilkes Done, and looked into the correct response to use for delete (with no content) and most say to use 204 response. So have updated to use that.

**Review by JoshuaWilkes — APPROVED** [review]

**Pull request merged**